app/code/community/SomethingDigital/Modal/etc/config.xml

```diff
@@ -36,6 +36,7 @@
                 <skip_cookie_check>0</skip_cookie_check>
                 <error_message>There was an error submitting the form. Please try again.</error_message>
                 <skip_if_already_subscribed>1</skip_if_already_subscribed>
+                <skip_if_already_logged_in>1</skip_if_already_logged_in>
```

Comment:
Let's just remove the default.

BC = backwards compatibility.  Maybe someone already using SomethingDigital_Modal likes how it applies to logged in users.  Changing the default will affect them too, maybe surprisingly.